Add Double Word Port IO

rianquinn opened the pull request:
For some reason we only have byte and word, and we were missing
double word for portio, so this adds the missing API.

Signed-off-by: “Rian <“rianquinn@gmail.com”>

connojd commented on bfvmm/include/intrinsics/portio_x64.h:
```diff
@@ -39,13 +41,17 @@ namespace portio
     auto inw(P port) noexcept { return __inw(gsl::narrow_cast<port_addr_type>(port)); }
 
+    template<class P, class = typename std::enable_if<std::is_integral<P>::value>::type>
+    auto inq(P port) noexcept { return __inq(gsl::narrow_cast<port_addr_type>(port)); }
```
Should this be ind or inl instead of inq? I thought q usually stood for 8 bytes.

rianquinn commented:
@brendank310 / @connojd I changed "q" -> "d". I also added the string versions, and their rep prefix versions. Take a quick look at the assembly and let me know if it makes sense. Once the tests pass I will ping you guys.

rianquinn commented:
@brendank310 / @connojd passed, ready to merge
